hmpy/views/gauge.py

```diff
@@ -54,12 +58,14 @@ def value(self, value):
         """
         if value < self.__min_value:
             self.__value = self.__min_value
```

Comment:
It seems like we are still limiting the actual value to the min or max value, not sure what we are actually accomplishing with this change